packages/core-data/src/entities.js

```diff
@@ -14,10 +14,13 @@ import apiFetch from '@wordpress/api-fetch';
 import { getEntitiesByKind } from './selectors';
 import { addEntities } from './actions';
 
+const getEditContext = () => 'edit';
+const getMediaContext = ( state ) => state.hasUploadPermissions ? 'edit' : 'view';
```

Comment:
This code needs some work. If anyone has an ideas on how we can make entities 'permissions aware', that would be great!